public_html/wp-content/themes/pattern-directory/src/components/active-filter-bar/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { select } from '@wordpress/data';
+import { store } from '@wordpress/viewport';
+import { Flex, FlexItem } from '@wordpress/components';
+
+function ActiveFilterBar( { children, actionsTitle, actions } ) {
+	const isSmall = select( store ).isViewportMatch( '< medium' );
+
+	return (
+		<Flex className="active-filter__bar" align="center">
+			<FlexItem className="active-filter__bar__copy">{ children }</FlexItem>
+			{ ! isSmall && actions.length > 0 && (
+				<FlexItem>
+					<Flex gap={ 0 }>
+						<FlexItem>
+							<h3 className="active-filter__bar__title">{ actionsTitle } </h3>
+						</FlexItem>
+
+						<FlexItem>
+							<ul>
+								{ actions.map( ( i ) => (
+									<li key={ i.href }>
+										<a href={ i.href }>{ i.label }</a>
```

Comment:
The example showed the `href` values like `#columns`; would clicking on that trigger a change in the active filters and results?

If so, it seems like `<button>` is more semantically appropriate.

Reply:
Since this is a navigation change, it should be a link. In production these will be category (or keyword) archives, ex: `https://wordpress.org/patterns/pattern-categories/columns/`

Reply:
👍

Reply:
Ah, I didn't realize it was changing the URL, that's much better :)